proto/internalapi/scanner/v4/common.proto

```diff
@@ -0,0 +1,58 @@
+// This contains protobuf types in pair with ClairCore's types, with
+// minimal differences. See https://github.com/quay/claircore for comments
+// on the fields.
+
+syntax = "proto3";
+
+option go_package = "v4";
```

Comment:
do we not need to define the proto package here?

Reply:
I think we do. Not sure what you're suggesting.